### PR TITLE
Rollup

### PR DESCRIPTION
**Pull request by DrRataplan**
**Also includes the function name resolver**

* Function name resolver
* A simple rollup config that basically just bundles all relevant code into one file and combines imports (todo: test it)
Add a github actions config for CI and to verify the build keeps working
TODO: Add the other check scripts, like lint. But let's first fix the warnings/errors:
```
✖ 11100 problems (9258 errors, 1842 warnings)
  1838 errors and 0 warnings potentially fixable with the `--fix` option.
```
:grimacing:

### CODE REVIEW

**Comment by DrRataplan** [comment]
The built file looks like this:

```js
import { evaluateXPath, registerCustomXPathFunction, registerXQueryModule, evaluateXPathToFirstNode, evaluateXPathToNodes, evaluateXPathToBoolean, parseScript, evaluateXPathToString } from 'fontoxpath';
import { LitElement, css as css$1, html as html$2 } from 'lit-element';

/**
 * Class for holding ModelItem facets.
 *
 * A ModelItem annotates nodes that are referred by a fx-bind element with facets for calculation and validation.
 *
 * Each bound node in an instance has exactly one ModelItem associated with it.
 */
class ModelItem {

```
... dozens of classes ...
```js
window.customElements.define('fx-message', FxMessage);

/**
 * `fx-send` - finds and activates a `fx-submission` element.
 *
 *
 * @customElement
 */

class FxSend extends FxAction {
  static get properties() {
    return { ...super.properties,
      submission: {
        type: String
      }
    };
  }

  constructor() {
    super();
    this.value = "";
  }

  connectedCallback() {
    console.log('connectedCallback ', this);
    this.submission = this.getAttribute('submission');
  }

  execute() {
    super.execute();
    console.log('submitting ', this.submission);
    console.log('submitting model', this.getModel()); //if not exists signal error

    const submission = this.getModel().querySelector(`#${this.submission}`);

    if (submission === null) {
      this.dispatchEvent(new CustomEvent('error', {
        composed: true,
        bubbles: true,
        detail: {
          message: `fx-submission element with id: '${this.submission}' not found`
        }
      }));
    }

    console.log('submission', submission);
    submission.submit(); //if not of type fx-submission signal error
  }

}
window.customElements.define('fx-send', FxSend);
//# sourceMappingURL=bundle.js.map
```

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@jinntec/fore",
   "version": "0.11.1",
   "description": "Fore - Forms for the Web",
-  "module": "./index.js",
```
Comment (JoernT) [inline package.json:5]: not sure here - doesn't that mean that i publish the bundle to npm when doing npm publish? I'd like to keep the unbundled npm package (as: optimization is application responsibility)

Comment (DrRataplan) [reply package.json:5]: Agreed, I'll add it to the files instead

Comment (DrRataplan) [reply package.json]: Done, it is added to "files"

**Pull request merged**